third_party/nvidia/include/Dialect/NVGPU/IR/NVGPUOps.td

```diff
@@ -142,7 +142,7 @@ def NVGPU_LoadMatrixOp : NVGPU_Op<"ldmatrix", [MemoryEffects<[MemRead]>]> {
     ins LLVM_PointerShared:$addr,
     UnitAttr:$trans
   );
-  let results = (outs LLVM_AnyStruct:$result);
+  let results = (outs AnyTypeOf<[LLVM_AnyStruct, I32]>:$result);
```

Comment:
llvm was complaining that we were returning a struct with just one element from inline_asm...